src/name/ip_address.rs

```diff
@@ -178,16 +178,33 @@ impl<'a> IpAddressRef<'a> {
     }
 }
 
+#[cfg(feature = "std")]
+fn ipv6_to_uncompressed_string(octets: [u8; 16]) -> String {
+    octets
```

Comment:
This is doing a whole bunch of temporary allocations, which I don't think is a good idea.

Reply:
Addressed in https://github.com/rustls/webpki/pull/21.